code/game/striketeams/striketeam_datums.dm

```diff
@@ -0,0 +1,185 @@
+var/list/sent_strike_teams = list()
+
+/datum/striketeam
+	var/striketeam_name = "Spec.Ops."
+	var/faction_name = "Nanotrasen"
+	var/mission = "Clean up the Station of all enemies of Nanotrasen. Avoid damage to Nanotrasen assets, unless you judge it necessary."
+	var/team_size = 6
+	var/min_size_for_leader = 4//set to 0 so there's always a designated team leader or to -1 so there is no leader.
+	var/spawns_name = "Commando"
+	var/can_customize = FALSE
+	var/logo = "nano-logo"
+
+	var/list/applicants = list()
+	var/searching = FALSE
+
+	var/leader_key = ""
+	var/list/team_composition = list()
+
+
+/datum/striketeam/proc/trigger_strike(var/mob/user)
+	//Is the game started
+	if(!ticker)
+		if(user)
+			to_chat(user, "<span class='warning'>The game hasn't started yet!</span>")
+		qdel(src)
+		return
+
+	//Has someone already sent a strike team of this type
+	if(striketeam_name in sent_strike_teams)
+		if(user)
+			to_chat(user, "<span class='warning'>[faction_name] has already sent \a [striketeam_name].</span>")
+		qdel(src)
+		return
+
+	//Logging
+	message_admins("<span class='notice'>[key_name(user)] is preparing a [striketeam_name].</span>", 1)
+
+	if(user)
+		if(alert("Do you really want [faction_name] to send in the [striketeam_name]?",,"Yes","No")!="Yes")
+			qdel(src)
+			return
+
+		mission = input(user, "Please specify which mission the [striketeam_name] shall undertake.", "Specify Mission", "")
+
+		if(!mission)
+			if(alert("Error, no mission set. Do you want to exit the setup process?",,"Yes","No")=="Yes")
+				qdel(src)
+				return
+			else
+				mission = initial(mission)
+
+		if(striketeam_name in sent_strike_teams)
+			to_chat(user, "Looks like someone beat you to it.")
+			qdel(src)
+			return
+
+	sent_strike_teams |= striketeam_name
+	sent_strike_teams[striketeam_name] = src
+
+	if(user)
+		to_chat(user, "<span class='notice'>[faction_name] has received your request. Commando applications will be open for the next minute.</span>")
+
+	searching = TRUE
+
+	var/icon/team_logo = icon('icons/mob/mob.dmi', logo)
+	for(var/mob/dead/observer/O in dead_mob_list)
+		if(jobban_isbanned(O, "Strike Team") || O.client.is_afk())
+			continue
+
+		to_chat(O, "[bicon(team_logo)]<span class='recruit'>[faction_name] needs YOU to become part of its upcoming [striketeam_name]. (<a href='?src=\ref[src];signup=\ref[O]'>Apply now!</a>)</span>[bicon(team_logo)]")
+
+	spawn(600)
+		searching = FALSE
+
+		for(var/mob/dead/observer/O in dead_mob_list)
+			if(jobban_isbanned(O, "Strike Team") || O.client.is_afk())
+				continue
+			to_chat(O, "[bicon(team_logo)]<span class='recruit'>Applications for [faction_name]'s [striketeam_name] are now closed.</span>[bicon(team_logo)]")
+
+		if(!applicants || applicants.len <= 0)
+			if(user)
+				to_chat(user, "<span class='warning'>[faction_name] received no applications for the [striketeam_name].</span>")
```

Comment:
This should probably be logged to all admins because if the admin starting this logs out, the error goes on silent ears.

Also using warning which is used a lot IC seems like too easily missed.